### ISSUE TITLE
List Process

### ISSUE THREAD

**Issue by pablomelo** [nuveo/gofn#43]
As the Admin,
I need to see a list of the containers that were started by gofn,
So I can have an overview of the state of my services.

**Comment by avelino** [comment]
I don't think it would be the responsibility of the gofn manage processes, we can create the gofn-server

**Comment by crgimenes** [comment]
Keeping the list in gofn is an easy way to access the container ID without having to change the signature of the functions or change the current code too much. This list would make very easy to implement some things like kill container for example. 
What we really need is the ID of the container.

**Comment by felipeweb** [comment]
Not only the container ID, but the machine ID too because sometimes shit happens and we need kill the machine

**Comment by Horaddrim** [comment]
I think a list of what containers gofn rolled up in the host it's cool, just for managing as crgimenes sad :+1:

**Comment by avelino** [comment]
I see this feature with a function of a CLI[1], example:
```
λ ~/ gofn ps
CONTAINER ID        IMAGE               COMMAND             CREATED             STATUS              PORTS               NAMES
```

bind to docker CLI, with filter to bring only that was run via gofn (not all containers)

[1] https://github.com/gofn/cli

**Comment by Horaddrim** [comment]
I've imagined exactly the same! hahahaha, I'll do it :smile: And really thanks for this lightning response!

**Comment by avelino** [comment]
@Horaddrim gofn is a project that animates me very much, unfortunately I walk without time to return contribute. I appreciate your interest in contributing to the project, I'm here to help

**Comment by Horaddrim** [comment]
Do you think it's a good time to use `dep` in the project? I'm working on this `ps` command right now, and I catch myself thinking about it :thinking:

**Comment by Horaddrim** [comment]
Really glad to help you guys! :smile:, there is anything more we can upgrade about this list feature?

**Comment by felipeweb** [comment]
Maybe add command on CLI
